qiskit_ibm_runtime/api/session.py

```diff
@@ -139,7 +139,6 @@ class RetrySession(Session):
     def __init__(
         self,
         base_url: str,
-        access_token: Optional[str] = None,
```

Comment:
No need to pass an access token parameter because authentication is transparently handled via the new authentication handlers (`LegacyAuth, CloudAuth`). 

Related links
- https://docs.python-requests.org/en/master/user/authentication/#new-forms-of-authentication